### ISSUE TITLE
 attempt to index global 'PVPFrame' (a nil value)

### ISSUE THREAD

**Issue by Zoobug** [Pridity/Lorti-UI-Classic#7]
Hi. Lua error when i open the Classic Season of Mastery please see below.

Message: Interface\AddOns\Lorti-UI-Classic\core\frames.lua:491: attempt to index global 'PVPFrame' (a nil value)
Time: Wed Nov 24 09:41:50 2021
Count: 1
Stack: Interface\AddOns\Lorti-UI-Classic\core\frames.lua:491: attempt to index global 'PVPFrame' (a nil value)
[string "@Interface\AddOns\Lorti-UI-Classic\core\frames.lua"]:491: in main chunk

Locals: addon = "Lorti-UI-Classic"
ns = <table> {
 cfg = <unnamed> {
 }
}
cfg = <unnamed> {
 textures = <table> {
 }
 consolidatedTooltipScale = 1.200000
 color = <table> {
 }
 buffFrame = <table> {
 }
 hotkeys = <table> {
 }
 combineBuffsAndDebuffs = false
 itemcount = <table> {
 }
 font = "Fonts\FRIZQT__.TTF"
 debuffFrame = <table> {
 }
 adjustOneletterAbbrev = true
 0 = <userdata>
 background = <table> {
 }
 cooldown = <table> {
 }
 macroname = <table> {
 }
}
dragFrameList = nil
event_frame = <unnamed> {
 0 = <userdata>
}
errormessage_blocks = <table> {
 1 = "Способность пока недоступна"
 2 = "Выполняется другое действие"
 3 = "Невозможно делать это на ходу"
 4 = "Предмет пока недоступен"
 5 = "Недостаточно"
 6 = "Некого атаковать"
 7 = "Заклинание пока недоступно"
 8 = "У вас нет цели"
 9 = "Вы пока не можете этого сделать"
 10 = "Ability is not ready yet"
 11 = "Another action is in progress"
 12 = "Can't attack while mounted"
 13 = "Can't do that while moving"
 14 = "Item is not ready yet"
 15 = "Not enough"
 16 = "Nothing to attack"
 17 = "Spell is not ready yet"
 18 = "You have no target"
 19 = "You can't do that yet"
}
enable = <function> defined @Interface\AddOns\Lorti-UI-Classic\core\frames.lua:43
onevent = <function> defined @Interface\AddOns\Lorti-UI-Classic\core\frames.lua:57
uierrorsframe_addmessage = <function> defined @Interface\AddOns\Lorti-UI-Classic\core\frames.lua:48
old_uierrosframe_addmessage = nil
CF = <unnamed> {
 0 = <userdata>
}
OnEvent = <function> defined @Interface\AddOns\Lorti-UI-Classic\core\frames.lua:275
f = <unnamed> {
 0 = <userdata>
}
a = BankPortraitTexture {
 0 = <userdata>
}
b = <unnamed> {
 0 = <userdata>
}
c = BankFrameTitleText {
 0 = <userdata>
}
d = <unnamed> {
 0 = <userdata>
}
_ = <unnamed> {
 0 = <userdata>
}
e = nil
a = <unnamed> {
 0 = <userdata>
}
b = <unnamed> {
 0 = <userdata>
}
c = <unnamed> {
 0 = <userdata>
}
d = <unnamed> {
 0 = <userdata>
}
_ = CharacterLevelText {
 0 = <userdata>
}
e = CharacterTitleText {
 0 = <userdata>
}
a = <unnamed> {
 0 = <userdata>
}
b = <unnamed> {
 0 = <userdata>
}
c = <unnamed> {
 0 = <userdata>
}
d = <unnamed> {
 0 = <userdata>
}
a = <unnamed> {
 0 = <userdata>
}
b = <unnamed> {
 0 = <userdata>
}
c = <unnamed> {
 0 = <userdata>
}
d = <unnamed> {
 0 = <userdata>
}
(*temporary) = nil
(*temporary) = nil
(*temporary) = 4
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = <unnamed> {
 0 = <userdata>
}
(*temporary) = 0.350000
(*temporary) = 0.350000
(*temporary) = 0.350000
(*temporary) = <userdata>
(*temporary) = ContainerFrame4BackgroundBottom {
 0 = <userdata>
}
(*temporary) = ContainerFrame5BackgroundTop {
 0 = <userdata>
}
(*temporary) = ContainerFrame5BackgroundMiddle1 {
 0 = <userdata>
}
(*temporary) = ContainerFrame5BackgroundBottom {
 0 = <userdata>
}
(*temporary) = ContainerFrame6BackgroundTop {
 0 = <userdata>
}
(*temporary) = ContainerFrame6BackgroundMiddle1 {
 0 = <userdata>
}
(*temporary) = ContainerFrame6BackgroundBottom {
 0 = <userdata>
}
(*temporary) = ContainerFrame7BackgroundTop {
 0 = <userdata>
}
(*temporary) = ContainerFrame7BackgroundMiddle1 {
 0 = <userdata>
}
(*temporary) = ContainerFrame7BackgroundBottom {
 0 = <userdata>
}
(*temporary) = ContainerFrame8BackgroundTop {
 0 = <userdata>
}
(*temporary) = ContainerFrame8BackgroundMiddle1 {
 0 = <userdata>
}
(*temporary) = ContainerFrame8BackgroundBottom {
 0 = <userdata>
}
(*temporary) = ContainerFrame9BackgroundTop {
 0 = <userdata>
}
(*temporary) = ContainerFrame9BackgroundMiddle1 {
 0 = <userdata>
}
(*temporary) = ContainerFrame9BackgroundBottom {
 0 = <userdata>
}
(*temporary) = ContainerFrame10BackgroundTop {

**Comment by Sukhumi** [comment]
Yeah this is the TBC version, where the PvP tab shows arena and stuff. I can whip you up a version that would work with SoM if there are none available.

**Comment by Zoobug** [comment]
> 

I've not found a SoM version so it would be great it if you could.

**Comment by Sukhumi** [comment]
https://github.com/Sukhumi/Lorti-UI-Classic/tree/Lorti-SOM this should work for SOM
